casa.js

```diff
@@ -56,8 +56,17 @@ function CasaBootstrap(expressApp, config) {
   ];
   const I18nUtility = I18n(localeDirs, config.i18n.locales);
 
-  // Mount all pre-journey middleware
-  casa.mountCommonExpressMiddleware(expressSession, expressJs.static, I18nUtility);
+  // Pre-journey middleware
+  const commonMiddlewareArgs = [casa, expressSession, expressJs.static, I18nUtility];
```

Comment:
Suggest not binding `this` to anything just to keep the returned `mountCommonMiddleware ` function as single-minded as possible!
```suggestion
const commonMiddlewareArgs = [null, expressSession, expressJs.static, I18nUtility];
```

Reply:
Sorry, we need to in this one place.

Because existing code already destructures `{ config }` from `this` here:
https://github.com/dwp/govuk-casa/blob/master/lib/Casa.js#L229

Reply:
Missed that 👍